data/actions/lib/actions.lua

```diff
@@ -1,20 +1,71 @@
-local wildGrowth = {1499, 11099} -- wild growth destroyable by machete
+local wildGrowth = { 1499, 11099 } -- wild growth destroyable by machete
 local jungleGrass = { -- grass destroyable by machete
 	[2782] = 2781,
 	[3985] = 3984,
-	[19433] = 19431
+	[19433] = 19431,
 }
-local groundIds = {354, 355} -- pick usable ground
-local sandIds = {231, 9059} -- desert sand
+local groundIds = { 354, 355 } -- pick usable ground
+local sandIds = { 231, 9059 } -- desert sand
 local holeId = { -- usable rope holes, for rope spots see global.lua
-	294, 369, 370, 383, 392, 408, 409, 410, 427, 428, 429, 430, 462, 469, 470, 482,
-	484, 485, 489, 924, 1369, 3135, 3136, 4835, 4837, 7933, 7938, 8170, 8249, 8250,
-	8251, 8252, 8254, 8255, 8256, 8276, 8277, 8279, 8281, 8284, 8285, 8286, 8323,
-	8567, 8585, 8595, 8596, 8972, 9606, 9625, 13190, 14461, 19519, 21536, 23713,
-	26020
+	294,
```

Comment:
noooooooooo, one id per line is ugly

Reply:
There must be other formatting styles that can be applied, I have no doubt! (I agree that this is very ugly).